### ISSUE TITLE
After removing Zen mods, some settings from the extension remain active in the browser

### ISSUE THREAD

**Issue by utopiaeh** [zen-browser/desktop#3837]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

After removing Zen mods, some settings from the extension remain active in the browser, even after deleting the mod. However, it seems that only a few of them are still active.

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.2

### What platform are you seeing the problem on?

macOS - aarch64

**Comment by djorium** [comment]
Which ones, please provide more detail.

**Comment by utopiaeh** [comment]
> Which ones, please provide more detail.

https://zen-browser.app/mods/ad97bb70-0066-4e42-9b5f-173a5e42c6fc/

I don’t remember the exact combination, but if I disable the workspace icon and then remove the mod, the icon remains hidden. I had to download it again and re-enable it.

https://zen-browser.app/mods/32aca67a-ffdd-49e7-95c7-1821793610ca/
Also, some issues were found with mods that manipulate the status bar. After removing the mods, the status bar remained floating, along with settings I had modified, like margin and offset. However, after downloading and resetting to the default values, everything reverted to normal.
